satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -615,6 +615,31 @@ read all (
 	where storagenode_storage_tally.interval_end_time >= ?
 )
 
+//--- peer_identity ---//
+
+model peer_identity (
+    key serial_number 
+
+    index (
+        name serial_number_update_at
+        fields serial_number update_at 
+    )
+    
+    field serial_number  blob
```

Comment:
Just curious, why we care about the serial number?

Reply:
serial number is unique and had to use to make multiple entries for same nodeID